change/@fluentui-web-components-7b17a2b2-f134-4934-8331-709bea980b6c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=32116&commitId=0c61d1fe4a4390c84fe185024557b6beff7f30c0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Beak 25.chromium.png|2186| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.Right top edge.chromium.png|1127| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>